.github/workflows/ci.yaml

```diff
@@ -16,6 +16,8 @@ jobs:
             toxenv: flake8
           - python: 3.x
             toxenv: py3flake8
+          - python: 3.x
```

Comment:
Can we package all of our linting tools into one place? Easier for contributors (just run `tox -e lint`) and likely faster for GHA.

Reply:
I like it! Done in latest.